hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -3,6 +3,9 @@
 // SPDX-License-Identifier: Apache-2.0
 { name: "entropy_src",
   clock_primary: "clk_i",
+  reset_primary: "rst_ni",
+  other_clock_list: ["clk_es_50mhz_i"]      
```

Comment:
@mwbranstad what is the goal for this clock? `clk_i` is typically the high speed jittery (read: secure) clock. Do you need a slower one too? @eunchan or @tjaychen might be able to give you the appropriate name depending upon your needs here.

Reply:
@sjgitty this is the same clock that is going to the Nuvoton AST logic, as defined by them. The email describing the provided entropy interface will produce 8 bits of entropy on every 50 MHz clock tick. I picked this name, somewhat following the convention of the usbdev clocking, but changed from 48 MHz to 50 MHz. Will use any recommended clock name.

Reply:
I actually think you don't need 2 clocks here. I think the intent of Nuvoton is that they'll sync to whatever clock domain you are using to capture. So I think if your module is running on `clk_i` (whatever it is), we'll likely feed that clock to them so they'll sample to it also.

Reply:
I was assuming Nuvoton would take in the same clk_es_50mhz_i clock. According to Ziv's email, they can generate entropy at a 50 mhz rate. I was also assuming that clk_i is the TLUL clock running at 100-200 mhz, so it cannot be the same clock. If things have changed, or you have newer information, I would like to see that.

Reply:
50mhz?? that seems way too high.  
But to clarify, Ziv's point is not that entropy will actually come out at that speed, but rather the interface between the analog module and entropy_src will be synchronous to a particular clock.

So for example his interface can be synchronous to a 100MHz clock, but the samples only change at a lower rate.  So the idea is that entropy_src could just directly sample the line at the lower rate without worrying about CDC.

If it helps, I can setup a meeting between all of us to go over any additional points.

Reply:
Yes, a meeting would be good to flatten this interface. Because there was no "valid" signal in Ziv's interface, I would have to infer then that all clock cycles have valid data, so I put the async i/f there. It sounds like this may not be needed, which is fine with me, but I currently need an interface defined that I can design to.